ta_services/secure_key_services/include/sks_ta.h

```diff
@@ -14,13 +14,615 @@
+ * SKS_CMD_DECRYPT_INIT - Initialize encryption processing
+ *
+ * param#0: in(*)-memref : [uint32_t session_handle]
+ *			   [struct sks_reference proc + proc parameters data]
+ * param#1: none
+ * param#2: none
+ * param#3: none
+ *
+ * These commands relate to the PKCS#11 API functions C_EncryptInit() and
+ * C_DecryptInit.
+ */
+#define SKS_CMD_ENCRYPT_INIT		0x00000010
+#define SKS_CMD_DECRYPT_INIT		0x00000011
+
+/*
+ * SKS_CMD_ENCRYPT_UPDATE - Update encryption processing
+ * SKS_CMD_DECRYPT_UPDATE - Update decryption processing
+ *
+ * param#0: in(*)-memref : [uint32_t session_handle]
+ * param#1: in-memref : [input-data]
+ * param#2: out-memref : [output-data]
+ * param#3: none
+ *
+ * These commands relate to the PKCS#11 API functions C_EncryptUpdate() and
+ * C_DecryptUpdate.
+ */
+#define SKS_CMD_ENCRYPT_UPDATE		0x00000012
+#define SKS_CMD_DECRYPT_UPDATE		0x00000013
+
+/*
+ * SKS_CMD_ENCRYPT_FINAL - Finalize encryption processing
+ * SKS_CMD_DECRYPT_FINAL - Finalize decryption processing
+ *
+ * param#0: in(*)-memref : [uint32_t session_handle]
+ * param#1: none
+ * param#2: out-memref : [output-data]
+ * param#3: none
+ *
+ * These commands relate to the PKCS#11 API functions C_EncryptFinal() and
+ * C_DecryptFinal.
+ */
+#define SKS_CMD_ENCRYPT_FINAL		0x00000014
+#define SKS_CMD_DECRYPT_FINAL		0x00000015
+
+/*
+ * SKS_CMD_GENERATE_SYMM_KEY - Generate a symmetric key
+ *
+ * param#0: in(*)-memref : [uint32_t session_handle]
+ *			   [struct sks_reference proc + proc parameters data]
+ *			   [struct sks_object_head attribs + attributes data]
+ * param#1: none
+ * param#2: out-memref : [uint32_t object_handle]
+ * param#3: none
+ *
+ * This command relates to the PKCS#11 API functions C_GenerateKey() and
+ * C_DecryptInit.
+ */
+#define SKS_CMD_GENERATE_SYMM_KEY	0x00000016
+
+/*
+ * SKS_CMD_SIGN_INIT - Initialize a signature computation processing
+ * SKS_CMD_VERIFY_INIT - Initialize a signature verification processing
+ *
+ * param#0: in(*)-memref : [uint32_t session_handle]
+ *			   [uint32_t key_handle]
+ *			   [struct sks_reference proc + proc parameters data]
+ * param#1: none
+ * param#2: none
+ * param#3: none
+ *
+ * These commands relate to the PKCS#11 API functions C_SignInit() and
+ * C_VerifyInit.
+ */
+#define SKS_CMD_SIGN_INIT		0x00000017
+#define SKS_CMD_VERIFY_INIT		0x00000018
+
+/*
+ * SKS_CMD_SIGN_UPDATE - Initialize a signature computation processing
+ * SKS_CMD_VERIFY_UPDATE - Initialize a signature verification processing
+ *
+ * param#0: in(*)-memref : [uint32_t session_handle]
+ * param#1: in-memref : [input-data]
+ * param#2: none
+ * param#3: none
+ *
+ * These commands relate to the PKCS#11 API functions C_SignUpdate() and
+ * C_VerifyUpdate.
+ */
+#define SKS_CMD_SIGN_UPDATE		0x00000019
+#define SKS_CMD_VERIFY_UPDATE		0x0000001a
+
+/*
+ * SKS_CMD_SIGN_FINAL - Initialize a signature computation processing
+ * SKS_CMD_VERIFY_FINAL - Initialize a signature verification processing
+ *
```

Comment:
s/Initialize/Finalize in description for both Sign and Verify

Reply:
thanks